### DISCUSSION TITLE
create-album-folder not working

### DISCUSSION THREAD

**Discussion by suprPHREAK** [simulot/immich-go#283] [General]
Hello, 
I'm importing a large amount of photos from a windows 10 machine over my network to my linux-based Immich server. The photos are all sorted into folders for the various events, I'd like those folders to become albums, which is what I uderstood "-create-album-folder" would do. I'm using the following command:

`immich-go -server=http://192.x.x.x:xxxx -key=xxxxxxxxxxx upload -create-album-folder c:\photos`

All of the folders are within the photos folder.

Running the command, it is importing all the files, but no albums are being created along the way. Am I misunderstanding the use case?

note: am total noob when it comes to  command line an linux

UPDATE: it made the albums at the end. Everything worked great!

**Comment by suprPHREAK** [comment]
Albums created after all files have been uploaded. Just have to be patient!

**Reply by simulot** [reply]
I need to change that
